.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,7 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Added or updated unit **and** E2E tests for my change.
+- [ ] Added or updated unit tests **and** checked with reviewers/maintainers whether E2E tests are needed for my change.
```

Comment:
```suggestion
- [ ] Added or updated unit tests for my change.
- [ ] Added or updated E2E tests for my change.
```

I'd prefer to break these out into two lines. This is partly because of aesthetics - I think the list looks cleaner when it doesn't wrap. More importantly though, this allows a reviewer to see at a glance whether the PR has unit tests _and_ E2E tests, or only one.

Perhaps we could add a sentence or two to the comment block above to _briefly_ explain on our expectations around unit and E2E tests. That keeps the list uncluttered and easy to read, but still gives folks guidance at a glance.